src/ctia/http/server.clj

```diff
@@ -153,18 +154,20 @@
            send-server-version]
     :or {access-control-allow-methods "get,post,put,patch,delete"
          send-server-version false}
-    :as http-config}]
+    :as http-config}
+   {{:keys [get-in-config]} :ConfigService
+     :as services} :- APIHandlerServices]
```

Comment:
`services` local is unused right now, but will be passed to almost all other routes in future PRs.